setup/install/providers/docker-registry.md

```diff
@@ -16,15 +16,24 @@ When configuring Docker Registries, an
 authenticate against a certain set of [Docker
 repositories](https://docs.docker.com/glossary/?term=repository){:target="\_blank"}.
 
+Perform the steps in this article in the same place where you have Halyard
+installed, whether [in a Docker
+container](/setup/install/halyard/#install-halyard-on-docker) or [locally on
+Ubuntu/Debian or macOS](/setup/install/halyard/#update-halyard-on-debianubuntu-or-macos).
+
 ## Prerequisites
 
-The Docker Registry you are configuring must already exist, support the [v2
-registry API](https://docs.docker.com/registry/spec/api/){:target="\_blank"}, and
-have at least 1 [tag](https://docs.docker.com/glossary/?term=tag){:target="\_blank"}
-among the repositories you define in your Account. While each different
-supported registry supports the same API, there are still some subtleties in
-getting them to work with Spinnaker. Please make sure you read the section below
-corresponding to your registry of choice:
+* The Docker Registry you are configuring must already exist.
+* That Registry must support the
+[v2 registry API](https://docs.docker.com/registry/spec/api/){:target="\_blank"}.
+* The Registry needs at least 1 [tag](https://docs.docker.com/glossary/?term=tag){:target="\_blank"}
```

Comment:
I think this only throws a warning, so this is more of a soft-requirement

Reply:
Halyard throws the warning?

Reply:
Yup, but it's non-blocking